test/e2e/cli/kubecli/kubecli.go

```diff
@@ -128,9 +128,20 @@ func CreateUserSecret(name, ns string) {
 	)
 	result := cli.GetSessionExitMsg(session)
 	EventuallyWithOffset(1, result).Should(SatisfyAny(Say(name+" created"), Say("already exists")), "Can't create user secret"+name)
+
+	labels := map[string]string{
+		"atlas.mongodb.com/type": "credentials",
```

Comment:
Since you added these values as const in `pkg/controller/connectionsecret/ensuresecret.go`, consider to use them instead of just strings here and down below.

Reply:
yes good idea, will do this.